<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindowClass</class>
 <widget class="QWidget" name="ChatWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>725</width>
    <height>550</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widgetLogo" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>751</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelLogo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>521</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../bitcoin.qrc">:/images/horLogo</pixmap>
     <pixmap>:/images/chatLogo</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="homepage" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QWidget" name="splitter" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>710</width>
       <height>600</height>
      </rect>
     </property>
     <widget class="QTabWidget" name="tab">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>550</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="chanServ">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="chat">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>510</y>
        <width>701</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="disconnect">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>480</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QListView" name="userView">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>30</y>
        <width>150</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="connectBox" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>566</width>
       <height>600</height>
      </rect>
     </property>
     <widget class="QLabel" name="logoTitle">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>16</y>
        <width>541</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>:/images/chatLogo</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="join">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Ubuntu&quot;; color: white;</string>
      </property>
      <property name="text">
       <string>Join LiteDoge IRC channel:</string>
      </property>
     </widget>
     <widget class="QLabel" name="nickname">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: italic 12pt &quot;Ubuntu&quot;; color: white;</string>
      </property>
      <property name="text">
       <string>Nickname</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPseudo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>180</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonConnect">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>220</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="text">
    <string>Fermer l'onglet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
